exercises/Ex8RemoveCharacter.js

```diff
@@ -1,7 +1,7 @@
 function removeCharacter(string , characterPosition){
```

Comment:
string is not a good name for the parameter - it's data type name. Avoid naming variables with reserved / keywords